Assets/Scripts/Coordinate.cs

```diff
@@ -17,12 +17,6 @@ public class Coordinate {
 	public override bool Equals(object coordinate)
 	{
 		Coordinate item = coordinate as Coordinate;
-
-		if (item == null)
-		{
-			return false;
-		}
-
-		return ( item.x == x & item.y == y);
+        return (item == null) ? false:( item.x == x & item.y == y);
```

Comment:
Spaces are a little off but fine I guess

Reply:
OK, @oguzkul.
I will take care from next time.
By the way thanks for pointing it out.